source/SpriteParameters.cpp

```diff
@@ -0,0 +1,158 @@
+/* SpriteParameters.cpp
+Copyright (c) 2024 by XessWaffle
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "SpriteParameters.h"
+
+#include "Sprite.h"
+
+
```

Comment:
```suggestion
using namespace std;


```
`std::get<>`s can probably stay in the current form, but I'd like to hear others' opinions

Reply:
`get<>` does feel a little ambiguous, I think for clarity's sake we keep it that way, but I don't really have a preference either way.

Reply:
keep it imo